### PR TITLE
Necessary files to run LERP

### PR DESCRIPTION
**Pull request by ommmid**
LERP is a linear interpolation (in joint space) motion planner which is created for tutorial purpose (https://github.com/ros-planning/moveit_tutorials/pull/331). In this PR, the necessary files corresponding to the lerp planner is added so we can run this planner for panda.

@davetcoleman @mlautman Please review.

### CODE REVIEW

File: .setup_assistant [hunk 1]
```diff
@@ -1,10 +0,0 @@
-moveit_setup_assistant_config:
```
Comment (rhaschke) [inline .setup_assistant]: Don't remove this file.

File: launch/demo.launch [hunk 2]
```diff
@@ -7,8 +7,13 @@
 
   <!-- By default, we are not in debug mode -->
   <arg name="debug" default="false" />
+<<<<<<< HEAD
```
Comment (rhaschke) [inline launch/demo.launch]: You pushed a merge conflict here.

File: launch/move_group_emptyplan.launch [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<launch>
```
Comment (rhaschke) [inline launch/move_group_emptyplan.launch]: What's the purpose of this file?

File: launch/run_benchmark_lerp.launch [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<launch>
```
Comment (rhaschke) [inline launch/run_benchmark_lerp.launch]: Do we really need to publish this file in this repo?

**Review by rhaschke — CHANGES_REQUESTED** [review]
Please have a look at the comments.

File: launch/demo.launch (hunk 2 above)
Comment (ommmid) [reply launch/demo.launch]: sorry! corrected it

File: launch/move_group_emptyplan.launch (hunk 3 above)
Comment (ommmid) [reply launch/move_group_emptyplan.launch]: we do not need it. removed it

File: launch/run_benchmark_lerp.launch (hunk 4 above)
Comment (ommmid) [reply launch/run_benchmark_lerp.launch]: LERP is just a simple planner, I guess we do not need benchmark for it. removed it.

File: config/panda_arm.xacro [hunk 5]
```diff
@@ -21,7 +21,7 @@
     </group_state>
     <!--END EFFECTOR: Purpose: Represent information about an end effector.-->
     <!--VIRTUAL JOINT: Purpose: this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
-    <virtual_joint name="virtual_joint" type="floating" parent_frame="world" child_link="panda_link0" />
+    <virtual_joint name="virtual_joint" type="fixed" parent_frame="world" child_link="panda_link0" />
```
Comment (rhaschke) [inline config/panda_arm.xacro]: This shouldn't be changed.

**Review by rhaschke — COMMENTED** [review]
I suggest not to clutter this PR with `moveit.rviz` changes. Otherwise looks good.

**Review by rhaschke — APPROVED** [review]

**Review by davetcoleman — APPROVED** [review]

**Comment by davetcoleman** [comment]
Thanks!

**Pull request merged**